lib/packwerk/commands/lazy_loaded_entry.rb

```diff
@@ -0,0 +1,37 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Packwerk
+  module Commands
+    class LazyLoadedEntry
```

Comment:
```suggestion
    class Entry
```
Nit: I think `LazyLoadedEntry` is a little too verbose.